Multisig/UI/Transaction/ExecuteTransaction/SecondaryDetailDisclosureCell.swift

```diff
@@ -16,7 +16,6 @@ class SecondaryDetailDisclosureCell: UITableViewCell {
         super.awakeFromNib()
         cellLabel.setStyle(.headline)
         self.backgroundView = UIView()
-        setBackgroundColor(.separator)
```

Comment:
Are you sure we don't need this set ?

Reply:
I didn't remove this. I have no idea. How do I find out?

Reply:
Changed to backgroundPrimary. Not sure how to verify this.